src/components/uploader/js/Uploader.js

```diff
@@ -206,7 +207,13 @@ var fluid_2_0_0 = fluid_2_0_0 || {};
         var numFilesInBatch = batch.files.length;
         var fileLabelStr = fluid.uploader.fileOrFiles(that, numFilesInBatch);
 
-        var uploadingSize = batch.totalBytesUploaded + that.queue.sizeOfUploadedFiles();
+        var uploadingSize = that.queue.sizeOfUploadedFiles();
+        if (previoussizeOfUploadedFiles !== uploadingSize){
+                        batch.totalBytesUploaded = 0;                             
```

Comment:
Indentation problem at this line, as well as the missing ; at line 215. Again, make sure to lint.